src/AppInstallerCLIE2ETests/TestHashHelper.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace AppInstallerCLIE2ETests
+{
+    public class TestHashHelper
+    {
+        public static string ExeInstallerHashValue { get; set; }
+
+        public static string MsiInstallerHashValue { get; set; }
+
+        public static string MsixInstallerHashValue { get; set; }
+
+        public static void HashInstallers()
+        {
+            ExeInstallerHashValue = HashInstallerFile(TestCommon.ExeInstallerPath);
+            //MsiInstallerHashValue = HashInstallerFile(TestCommon.MsiInstallerPath);
+            //MsixInstallerHashValue = HashInstallerFile(TestCommon.MsixInstallerPath);

```

Comment:
Why commented?

Reply:
Added if statement logic to deal with empty MSI/MSIX path parameter values.